src/main/java/com/rarchives/ripme/ripper/DownloadFileThread.java

```diff
@@ -90,7 +91,7 @@ public void run() {
 
                 // Setup HTTP request
                 HttpURLConnection huc;
-                if (this.url.toString().startsWith("https")) {
+                if (urlToDownload.toString().startsWith("https")) {
```

Comment:
Is this set somewhere besides `urlToDownload = this.url`? If not, this is basically just renaming the variable used, correct? Just want to make sure.